### PR TITLE
Add gradient Artist list and bug fixes

### PR DESCRIPTION
**Pull request by Lenkly**
Bugs fixed:
- Proxy
- mapping of genres (all put into arrays)
- gradient buttons for artists *

New: 
- Match Icon based on calculation of the user input (different colors)

 (*) As this isn't working 100% properly yet, this is only a draft request.

### CODE REVIEW

File: client/src/components/CalcIcon.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  font-size: 30px;
+  color: ${(props) => calcColor[props.color].textcolor};
+  background: none;
+  font-family: 'Rubik', sans-serif;
+  margin-bottom: 25px;
+`;
+
+const calcColor = {
+  perfect: { bordercolor: '#ff0084', textcolor: '#ff0084' },
+  veryhigh: { bordercolor: '#ff00f2', textcolor: '#ff00f2' },
```
Comment (lmachens) [inline client/src/components/CalcIcon.js:18]: camelCase in this file for all properties

```suggestion
  veryHigh: { borderColor: '#ff00f2', textColor: '#ff00f2' },
```

File: client/src/components/Festivals.js [hunk 2]
```diff
@@ -25,22 +32,44 @@ const FestivalIntroduction = styled.div`
...
+  overflow: hidden;
+`;
+const Fill = styled.div`
+  flex-grow: 1;
+  background: #f9f9f9;
 `;
 
 async function fetchFestivals() {
-  const response = await fetch('http://localhost:4000/api/festivals');
+  const response = await fetch('/festivals');
```
Comment (lmachens) [inline client/src/components/Festivals.js:72]: Make sure the API runs in `/api`. You might have to adjust your server (API) too. 

```suggestion
  const response = await fetch('/api/festivals');
```

File: package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
     "dev:server": "npm run build && cd .. && npm start",
     "server": "nodemon index.js",
     "start": "node index.js",
-    "api": "json-server --watch db.json --port 4000"
+    "api": "json-server --watch db.json --port 8080"
```
Comment (lmachens) [inline package.json:42]: You don't need this script if you load `json-server` in your index.js

File: client/src/components/Festivals.js (hunk 2 above)
Comment (Lenkly) [reply client/src/components/Festivals.js:72]: Whenever I set it to '/api/festivals' I get a 404.

**Pull request merged**